spec/models/form_spec.rb

```diff
@@ -387,10 +387,6 @@ def answer_local_authority(lettings_log)
       expect(form.sections[0].class).to eq(Form::Sales::Sections::Setup)
       expect(form.subsections.count).to eq(1)
       expect(form.subsections.first.id).to eq("setup")
-      expect(form.pages.count).to eq(13)
-      expect(form.pages.first.id).to eq("owning_organisation")
-      expect(form.questions.count).to eq(14)
-      expect(form.questions.first.id).to eq("owning_organisation_id")
```

Comment:
These are just testing properties of the sales setup section, which doesn't seem useful here.